plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -233,20 +245,22 @@ private AzureStorageSettings(
     }
 
     private AzureStorageSettings(
-        String account,
-        String key,
-        String sasToken,
-        String endpointSuffix,
-        TimeValue timeout,
-        int maxRetries,
-        TimeValue connectTimeout,
-        TimeValue writeTimeout,
-        TimeValue readTimeout,
-        TimeValue responseTimeout,
-        ProxySettings proxySettings
+            String account,
```

Comment:
Indentation change here intentional? Or are our IntelliJ rules different to the existing standard of this project?

Reply:
That's weird. Looks like my base branch is not up to date. I will rebase and force push it after other comments are marked as resolved.   

<img width="734" alt="Screenshot 2024-03-06 at 1 50 21 pm" src="https://github.com/chengwushi-netapp/OpenSearch/assets/153049940/c4649290-3ba8-4ef6-b8cb-84653230acaf">

Reply:
fixed